### ISSUE TITLE
Inability to pause gif on goose recoreded page

### ISSUE THREAD

**Issue by CKrawczyk** [ministryofjustice/record-a-goose-sighting#7]
The inability to stop or pause the goose gif on the last page is also an accessibility issue.  It breaks the [WCAG 2.2.2 Pause, Stop, Hide (Level A) Guideline](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#time-limits-pause) as the animation lasts more than 5 seconds.

**Comment by jbuller** [comment]
Great point! So question is, should we add this to the answers sheet as an issue to be detected, or remove it
 I vote for the former.

**Comment by CKrawczyk** [comment]
My vote is for adding it to the answers sheet, this is an issue sites get wrong.

**Comment by BNewing** [comment]
Good catch! I like the idea of adding it to the answers sheet. Do you have any text you'd suggest :)? Otherwise I'm happy to make a PR :)

**Comment by jbuller** [comment]
"Animation cannot be stopped. Moving content such as the happy goose can be a severe distraction for people with conditions such as attention deficit disorders, making it difficult to use the rest of the page. It should run no longer than 5 seconds or be able to be paused or hidden by the users."

**Comment by BNewing** [comment]
Thank you @jbuller - added in
